orbit/pkg/table/user_login_settings/user_login_settings.go

```diff
@@ -31,7 +32,7 @@ func Generate(ctx context.Context, queryContext table.QueryContext) ([]map[strin
 
 	ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
 	defer cancel()
-	cmd := exec.CommandContext(ctx, "dscl", ".", "-list", "/Users", "hint")
+	cmd := exec.CommandContext(ctx, "/usr/bin/dscl", ".", "-list", "/Users", "hint")
```

Comment:
I believe it's not an issue, but it's always good practice to use absolute paths when executing commands.